### PR TITLE
Added 'Save as Draft' option and removal of 'All registered zubhub users' option

### LINKED ISSUE #592: Behavior of “for all registered Zubhub users” viewing mode is not defined correctly.
### **The Bug**
When a creator sets the option to “for all registered Zubhub users” then an additional field opens to input usernames.

### **To Reproduce**

Visit: https://zubhub.unstructured.studio/
Login
Create a Project
Scroll down to the 8th option in the form.
Select “all registered zubhub users”.
An additional input field will appear.

### **Expected behavior**
The project should be visible to all the registered/authenticated users.
There shouldn't be any new input field to take in usernames.

### **Screenshots**

223364315-c48dd707-a616-43d9-ad53-64bdd0b852b2

### LINKED ISSUE #593: Behavior of “only a few zubhub users” viewing mode is not defined correctly.
### **The Bug**
When I set the option to “only a few zubhub users” it sends the project to draft, and it's only visible to me.

### **To Reproduce**
Visit: https://zubhub.unstructured.studio/
Login
Create a Project
Scroll down to the 8th option in the form.
Select the “only a few zubhub users” option.
The created project will be shifted to drafts.

### **Expected behavior**

The project shouldn't shift to drafts. It should be visible to selected zubhub users that a creator will select.

An additional input form should open when selecting “only a few zubhub users” during the project creation process, like the image below:
image

Screenshot of the bug
image
image

### PR DESCRIPTION
**Pull request by cherish2003**
## Summary
This PR renames the options as per operations 
The option `Only few zubhub users` is renamed to `Save as Draft` and `All registered zubhub users` is renamed to  `Only few zubhub users` this changes are done because  when we select option `All registered ZubHub` users it is showing for adding usernames but it is not possible to determine all the ZubHub users by taking as input and while we select option `Only few zubhub users` it saves the project to drafts so it would be perfect for renaming the options so the issue will be solved 

Closes #592, Closes #593
## Changes
- Renaming of options in translation.json file
## Screenshots
Screenshot 2023-03-16 at 3 16 17 PM

### CODE REVIEW

**Comment by cherish2003** [comment]
@tuxology can you kindly review the code and let me know any things are there to be done

**Comment by tuxology** [comment]
@cherish2003 Can you resolve conflicts?

File: zubhub_frontend/zubhub/src/assets/js/styles/views/create_project/createProjectStyles.js [hunk 1]
```diff
@@ -66,6 +67,22 @@ const styles = theme => ({
       },
     },
   },
+  circleBox: {
```
Comment (tuxology) [inline zubhub_frontend/zubhub/src/assets/js/styles/views/create_project/createProjectStyles.js:70]: What is the role of this?

**Comment by tuxology** [comment]
@cherish2003 Please resolve conflicts and push the changes. Also just one comment left here

**Comment by cherish2003** [comment]
@tuxology I will make another pull request with removing the unwanted changes

**Pull request closed without merging**